### DISCUSSION TITLE
boxed and wide layout at the same time

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#625] [Q&A]
I'm trying a mix of boxed layout with border and wide layout across the display width. The top bars section should cover the entire display width, the rest of the page should have a boxed layout. I can't find a solution. Does anyone have an idea how I can do this?
In the same way as in the picture.

![mixed-layout](https://github.com/templaza/astroid-framework/assets/157874871/646fda93-ff23-4323-9bc8-4d81908037f2)

**Comment by Chacapamac** [comment]
Can you procure the css code for these boxes?

**Maybe try changing the “Flexbox Alignments?**
https://mdbootstrap.com/docs/standard/layout/horizontal-alignment/

**Reply by ghost** [reply]
@Chacapamac 

Thank you very much for this tip. I've read a lot about it now, but I don't yet understand how to use it correctly. I will read more about it in the next few days to learn and understand it.

**Comment by ghost** [comment]
@chacapamac

I would like to thank you again for your tip. I have now read a lot about it and understand how it works. This brings me very close to my design idea.

**Comment by Chacapamac** [comment]
If I understand well you want the menu items to be equally separated on the total width of the website?

**Comment by ghost** [comment]
@Chacapamac 

No, that's not what I mean.
Unfortunately, English is not my mother tongue, so I find it very difficult to communicate in English. I apologize for my bad English.
I would like to have a boxed layout with border left and right. But the top section and the footer section should be shown over the whole display width.

![mixed-layout](https://github.com/templaza/astroid-framework/assets/157874871/81783f6d-8b3a-4228-9447-ba0aae9167f5)


I have found this:

`@media (min-width: 1200px){ .container, .container-sm, .container-md, .container-lg, .container-xl { max-width: 960px !important; } }`

I'm getting very close to my idea, but I can't get the border left and right as I want.

But that's not so bad, I like it even without border left and right.
And with Flexbox I can align everything in the container fluid the way I want.

Translated with DeepL.com (free version)
